Packs/ApiModules/Scripts/TAXII2ApiModule/TAXII2ApiModule.py

```diff
@@ -689,6 +695,114 @@ def parse_intrusion_set(self, intrusion_set_obj: Dict[str, Any]) -> List[Dict[st
         intrusion_set["fields"] = fields
         return [intrusion_set]
 
+    def parse_general_sco_indicator(
+        self, sco_object: Dict[str, Any], value_mapping: str = 'value'
+    ) -> List[Dict[str, Any]]:
+        """
+        Parses a single SCO indicator.
+
+        Args:
+            sco_object (dict): indicator as an observable object.
+            value_mapping (str): the key that extracts the value from the indicator response.
+        """
+        sco_indicator = {
+            'value': sco_object.get(value_mapping),
+            'score': 3,
+            'rawJSON': sco_object,
+            'type': STIX_2_TYPES_TO_CORTEX_TYPES.get(sco_object.get('type'))  # type: ignore[arg-type]
+        }
+
+        fields = {
+            'stixid': sco_object.get('id'),
+            'tags': list(set(self.tags))
+        }
+        if self.tlp_color:
+            fields['trafficlightprotocol'] = self.tlp_color
+        sco_indicator['fields'] = fields
+        return [sco_indicator]
```

Comment:
because if you take a look at the `parse_generator_type_envelope` and `parse_dict_envelope` function, you can see that it extends the results that it finds, hence it must be returned as a `list`. 
![image](https://user-images.githubusercontent.com/53861351/185414431-557de57f-39b5-47ef-9174-b9ea60c5bf78.png)

Reply:
also if you take a look at other indicator parsing, you will see they are returned as lists too